.github/workflows/actions.yml

```diff
@@ -0,0 +1,34 @@
+name: main
+
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  main-job:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - env:
+              TASK: lint
+            allow_failures: false
+            command: npm run lint
+
+          - env:
+              TASK: test
+            allow_failures: false
```

Comment:
What is the default value of this? sensible default IMO would be false, so if that is the case we can just remove these from here.

cc @shoukoo

Reply:
Ok had a chat with Andy just now, so `allow_failures` by itself does not do anything at all, its just a variable that we can use it within our workflow like this and it could be named anything since its variable,

```
...
continue-on-error: ${{ matrix.allow_failure }}
strategy:
  fail-fast: false
  matrix:
    include:
      - env:
           TASK: link
        allow_failure: true
....
```

For these tests, we can just remove allow_failure given we always want failed job to exit unsuccessfully. cc @token-cjg